pkgs/by-name/di/dita-ot/package.nix

```
@@ -0,0 +1,48 @@
+{ stdenv
+, fetchzip
+, bash
+, jdk17
+, nix
+, coreutils
+, lib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  name = "dita-ot";
+  version = "4.2.1";
+
+  buildInputs = [
+    bash
+    jdk17
+  ];
+
+  src = fetchzip {
+    url = "https://github.com/dita-ot/dita-ot/releases/download/${version}/dita-ot-${version}.zip";
+    hash = "sha256-PtscKsQwW1uBu0OfVHi7/d9JujGx12nZT+W3uMPQQWc=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/
+    cp -r $src/* $out/
+
+    runHook postInstall
+  '';
+
+  doInstallCheck = true;
+  installCheckPhase = ''
+    runHook preInstallCheck
+    $out/bin/dita --version
+    runHook postInstallCheck
+  '';
+
+  meta = {
+    homepage = "https://dita-ot.org";
+    changelog = "https://www.dita-ot.org/dev/release-notes/#v${version}"
+    description = "The open-source publishing engine for content authored in the Darwin Information Typing Architecture.";
```

```suggestion
    description = "The open-source publishing engine for content authored in the Darwin Information Typing Architecture";
```

Per [manual](https://nixos.org/manual/nixpkgs/stable/#var-meta-description).

---

Implemented 👍🏼